server/controller/Users/UserQueries.js

```diff
@@ -0,0 +1,46 @@
+const dbconnection = require('../../database/db_connection');
+
+const mysql = require("mysql");
+    // TODO: Global database config
+    const dbConfig = mysql.createConnection({
+        user: "b4bb330a3f009b",
+        host: "eu-cdbr-west-01.cleardb.com",
+        password: "4be56a74",
+        database: "heroku_1fcc54407741c0e",
+      });
+
+module.exports = {
+    register: (data, callBack) => {
+        dbConfig.query(
+            `insert into users(username, password) 
```

Comment:
please write the queries with the capitalize such as INSERT INTO ..... @nguyenquocviet2007